lisp/ess-sp4-d.el

```diff
@@ -34,11 +34,10 @@
 
```

Comment:
all *sp*.el files exept the most recent sp6 should go to /old.  I think sp6w is probably unnecessary as well. But who knows :(